examples/server.mjs

```diff
@@ -0,0 +1,44 @@
+import express from 'express';
+
+const app = express();
+const port = 3000;
+
+import * as dotenv from "dotenv";
+dotenv.config()
+
+import { load } from "@azure/app-configuration-provider";
+const connectionString = process.env.APPCONFIG_CONNECTION_STRING;
+const appConfig = await load(connectionString, {
+    featureFlagOptions: {
+        enabled: true,
+        selectors: [{
+            keyFilter: "*"
+        }],
+        refresh: {
+            enabled: true,
+            refreshIntervalInMs: 10_000
+        }
+    }
+});
+
+import { ConfigurationMapFeatureFlagProvider, FeatureManager } from "@microsoft/feature-management";
```

Comment:
Can you put the package import statement together at the beginning of the file?

Reply:
I referenced the example in JS provider repo: https://github.com/Azure/AppConfiguration-JavaScriptProvider/blob/main/examples/helloworld.mjs

Reply:
in JS repo, dotenv related imports are put aside on top of the file, only for loading connection string. And the rest of the sample demonstrates the required package to import and necessary statements to make it work. 

Here it's an express example, I would suggest either to follow JS provider to only put dotenv on top of the file, and other import statements together. Or just put them all together on top.

Reply:
I decide to remove dotenv and use connection string directly:

```
const connectionString = "<your-connection-string>"
```